model/src/main/proto/topic.proto

```diff
@@ -0,0 +1,98 @@
+syntax = "proto3";
+
+import "thumbnail.proto";
+
+package model;
+
+option java_package = "org.oppia.app.model";
+option java_multiple_files = true;
+
+// Corresponds to the on-disk storage representing all available topics for play.
+message Classroom {
+  // Known topics that the player can play.
+  TopicSummary topic_summary = 1;
+
+  // The last time this classroom was updated.
+  int64 last_update_time_ms = 2;
+}
+
+// Corresponds to the list of topics that can be shown on the homescreen.
+message TopicList {
+  // Corresponds to the story promoted at the top of the homescreen. Either the story is in-progress, or it's a
+  // recommended story if no other stories are in progress.
+  PromotedStory promoted_story = 1;
+
+  // All topics that are available to the player.
+  repeated TopicSummary topic_summary = 2;
+
+  // The total number of ongoing stories by the player.
+  int32 ongoing_story_count = 3;
+}
+
+// Corresponds to the list of stories the player is currently playing.
+message OngoingStoryList {
+  // Ongoing stories from within the last 7 days.
+  repeated PromotedStory recent_story = 1;
+
+  // Other ongoing stories from longer than 7 days ago.
+  repeated PromotedStory older_story = 2;
+}
+
+// The summary of a story that should be promoted, either because it's been started and not yet completed by the player,
+// or because they have completed all other lessons and may find this one interesting.
```

Comment:
Extra white space nit before `or because` from your last PR, same for other comments too.

Reply:
Ditto--this isn't a TODO.